### PR TITLE
added version specific websites #3459

### PR DESCRIPTION
**Pull request by udnisap**
I only implemented the version specific docs. What are we going to do with the default docs and how to navigate between docs?

### CODE REVIEW

**Comment by vjeux** [comment]
cc @foghina, @frantic

**Comment by udnisap** [comment]
This is as described in #3459 
cc @ide @mkonicek @dabit3

**Comment by mkonicek** [comment]
Cool! How did you test this?

File: website/publish.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 set -e
 
+VERSION=`git describe | sed 's/-.*//'`
```
Comment (mkonicek) [inline website/publish.sh]: What's the value of `VERSION` if we're on master?

Comment (mkonicek) [inline website/publish.sh]: I'm on branch 0.17-stable locally and this returns `v0.3.11`.

Comment (udnisap) [inline website/publish.sh]: sorry it should have been `git describe --abbrev=0 --tags`

**Comment by udnisap** [comment]
I sent initially to get some feedback as there were some clarifications.  
I didnt test the whole flow. I tested commenting out part of the shell script and running in my local. 
I commented out the `git rebase` as well.
Since there are docs in the current `gh-pages` branch this will have it. We might have to cleanup the repo and have a index manually. Shall i work on that as well?

**Comment by udnisap** [comment]
Is the bot having some issues? I did not update the pull request for a while

**Comment by satya164** [comment]
@udnisap Yeah. Ignore the bot.

**Comment by mkonicek** [comment]
`git describe` returns the version of git for me so I don't see how this code can work. Are you still interested in working on this?

**Comment by bestander** [comment]
Thanks for the contribution, @udnisap.
I am taking control of this feature today and I like your intention.

I'll close this PR though because the work will be coming from the internal codebase and will require some more back and forward review process. 
It affects the web server file structure after all.

**Pull request closed without merging**